core-tests/src/test/java/org/mule/runtime/core/routing/correlation/EventProcessingThreadTestCase.java

```diff
@@ -1,57 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-
-package org.mule.runtime.core.routing.correlation;
-
-import org.mule.runtime.core.routing.EventProcessingThread;
-import org.mule.tck.junit4.AbstractMuleTestCase;
-import org.mule.tck.probe.PollingProber;
-import org.mule.tck.probe.Probe;
-import org.mule.tck.probe.Prober;
-import org.mule.tck.size.SmallTest;
-
-import org.junit.Test;
-
-@SmallTest
-public class EventProcessingThreadTestCase extends AbstractMuleTestCase {
```

Comment:
It was used to test a generic threading support (https://github.com/mulesoft/mule/pull/4594/files#diff-9e058ba306685224ccdd94fa037340e7) that is now replaced with the Schedulers.